mod/pages/tests/phpunit/integration/Elgg/Pages/ElggPageIntegrationTest.php

```diff
@@ -28,6 +28,11 @@ class ElggPageIntegrationTest extends \Elgg\IntegrationTestCase {
 	 * {@inheritDoc}
 	 */
 	public function up() {
+		
+		if (!elgg_is_active_plugin('pages')) {
+			$this->markTestSkipped();
```

Comment:
Maybe should be done at a higher level during setUp. You can sniff plugin name, there is a method for it in one of parent classes